3-networks/envs/development/main.tf

```diff
@@ -105,7 +105,7 @@ module "restricted_shared_vpc" {
 }
 
 /******************************************
- Base shared VPC
```

Comment:
This should stay as Base as per page 30 of the security foundation guide here https://services.google.com/fh/files/misc/google-cloud-security-foundations-guide.pdf